src/app/review/review.component.ts

```diff
@@ -70,13 +90,63 @@ export class ReviewComponent implements OnInit {
                 this.coursesReviewed = Promise.resolve(coursesReviewed);
             });
             this.questions = this.reviewService.getQuestions();
-        } else { // Other user types
+        } else if(this.userType == 2){
+            this.user = this.teacherService.getTeacher(this.userID);
+            Promise.all([
+                this.courses = this.teacherService.getCoursesByTeacher(this.userID),
+                this.reviews = this.teacherService.getReviewsByTeacher(this.userID),
```

Comment:
we only need `this.reviews` here, `this.courses` can be removed.

Reply:
I would prefer to keep the course information with the review. The review only has the course id correct ?

Reply:
No it has some of the course information as well.

Here is an example:
``` json
{
    "course_name": "Software Engineering",
    "course_code": "CEN5035",
    "review_content": [
    {
        "answer": "Yeah they are great",
        "question": "Are they a good teacher",
        "label": "",
        "required": false,
        "order": 1,
        "controlType": "textbox",
        "type": ""
    },
    {
        "answer": "Nope",
        "question": "Any comments",
        "label": "",
        "required": false,
        "order": 1,
        "controlType": "textbox",
        "type": ""
    }
]
},
```